src/screens/change-wallet/ChangeWalletSheet.tsx

```diff
@@ -127,10 +127,10 @@ export default function ChangeWalletSheet() {
     const bluetoothWallets: AddressItem[] = [];
     const readOnlyWallets: AddressItem[] = [];
 
-    Object.values(walletsWithBalancesAndNames).forEach(wallet => {
+    Object.values(walletsWithBalancesAndNames)?.forEach(wallet => {
```

Comment:
This won't do anything because if `walletsWithBalancesAndNames` is undefined or null (all other types are handled by `Object.values`) then it will just throw an error. If it's possible that `walletsWithBalancesAndNames` is null or undefined then would have to do `Object.values(walletBalancesAndNames || {})`